### PR TITLE
Add a test for the stable/policy-limitmemory policy

### PR DESCRIPTION
**Pull request by mprahl**
Resolves:
https://github.com/stolostron/backlog/issues/20844

### CODE REVIEW

**Review by dhaiducek — DISMISSED** [review]
/hold (in case you want to dismiss the nits 😉 )

Looks good to me!

Nit: Should the name of the file match the name of the policy and file? (i.e. `policy-limitmemory_test.go`?)

**Comment by mprahl** [comment]
Thanks for the review @dhaiducek! The nits are picked.  :smile:

I imagine I'll have to rebase again shortly after #295 is merged.

**Review by dhaiducek — APPROVED** [review]

File: test/integration/policy-limitmemory_test.go
Comment (ycao56) [inline test/integration/policy-limitmemory_test.go:23]: Looks good. The only thing is I would suggest not to put BVT label on it. The config policy controller has been already tested by other test cases. It's safe to leave this test case for SVT.

File: test/integration/policy-limitmemory_test.go
Comment (ycao56) [inline test/integration/policy-limitmemory_test.go:103]:
```suggestion
				_, err := clientManaged.CoreV1().LimitRanges(userNamespace).Get(
```

File: test/integration/policy-limitmemory_test.go
Comment (ycao56) [inline test/integration/policy-limitmemory_test.go:120]:
```suggestion
		err = clientManaged.CoreV1().LimitRanges(userNamespace).Delete(
```

**Review by ycao56 — CHANGES_REQUESTED** [review]

File: test/integration/policy-limitmemory_test.go
Comment (mprahl) [reply test/integration/policy-limitmemory_test.go]: Thanks for catching that!

**Review by ycao56 — APPROVED** [review]

**Pull request merged**
